lib/FieldType/FieldTypeFormMapperInterface.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * This file is part of the eZ RepositoryFroms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+namespace EzSystems\RepositoryForms\FieldType;
+
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use Symfony\Component\Form\FormInterface;
+
+/**
+ * Interface for FieldType form mappers.
+ *
+ * FieldType form mappers makes it possible to "map" and adapt forms related to FieldTypes (e.g. FieldDefinition forms).
```

Comment:
What about this ?

"It maps a FieldType's specifics to its editing Forms" (editing, or something else, more specific)